src/main/java/software/amazon/msk/auth/iam/internals/MSKCredentialProvider.java

```diff
@@ -70,6 +74,8 @@ public class MSKCredentialProvider implements AWSCredentialsProvider, AutoClosea
     private static final Logger log = LoggerFactory.getLogger(MSKCredentialProvider.class);
     private static final String AWS_PROFILE_NAME_KEY = "awsProfileName";
     private static final String AWS_ROLE_ARN_KEY = "awsRoleArn";
+    private static final String AWS_ACCESS_KEY_ID = "awsAccessKeyId";
```

Comment:
Could we rename the keys to "awsRoleAccessKeyId" and "awsSecretAccessKey" to make it clear to the user that these credentials will be used to assume the role and not used directly?

Reply:
Sure, or could also add support for using them without assuming a role? I think that's the only other option that makes sense.

Reply:
Nevermind, I see the naming of awsRoleSessionName would be inconsistent otherwise. Adding key+secret without the role seems like a different use case so will not include it here.